src/problem/MFEMProblem.C

```diff
@@ -18,44 +19,71 @@ MFEMProblem::validParams()
   return params;
 }
 
+
 MFEMProblem::MFEMProblem(const InputParameters & params)
   : ExternalProblem(params),
-    _input_mesh(_mesh.parameters().get<MeshFileName>("file")),
-    _formulation(getParam<std::string>("formulation")),
```

Comment:
See above discussion on clang-format - also worth removing the commented out code here for _input_mesh.